### DISCUSSION TITLE
Why does XamlCodeGenerator fail to find type Prism.DryIoc.PrismApplication ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#11741] [Q&A]
The respective assembly is being referenced inside the project but type not found. 

<img width="626" alt="image" src="https://user-images.githubusercontent.com/43340959/226192629-e248e97c-9efd-4165-a562-5cc0c699ff23.png">

<img width="326" alt="image" src="https://user-images.githubusercontent.com/43340959/226192826-1b4bf0c1-6490-4718-b078-e4c8d3851042.png">

**Comment by Al3xMa** [comment]
Did you manage to solve it? I have similar problem, but with Toolkit.
![image](https://github.com/unoplatform/uno/assets/131765250/f0a5f243-ff30-426b-8294-653478b5009b)

![image](https://github.com/unoplatform/uno/assets/131765250/707e0528-b6b2-4417-ab68-a6a80a9d2693)

Problem appears when i add utu:AutoLayout in xaml.
![image](https://github.com/unoplatform/uno/assets/131765250/50941c20-0b63-4952-8a18-fa7d9f0b13e1)

I have tried clean, rebuild, restarted vs and nothing seems to help. 
Any idea what could it be?

**Reply by jeromelaban** [reply]
@RickyMartine This generally happens when you've named your project by ending it with `.Uno`. We generally try to handle all those cases, but we may have missed one. What's your Uno.WinUI package version?

**Reply by Al3xMa** [reply]
Yup, this was the problem. Thank you so much.
Version is 4.8.33.
